src/main.js

```diff
@@ -1,7 +1,14 @@
-import Vue from 'vue'
-import App from './App.vue'
+import Vue from 'vue';
+import App from './App.vue';
+import { Button, Dropdown, Select } from 'element-ui';
```

Comment:
Create a plugin with all global components, and we can import all the element ui components that we need there.